.travis.yml

```diff
@@ -6,6 +6,7 @@ branches:
   - master
 env:
   global:
+    - "SERVICE=ledger"
```

Comment:
hey why didn't we need this before?

Reply:
may be a relic from when i was debugging something. shall investigate

Reply:
this was so that utils tests that require configuration do not fail